iOS-NOTTODO/iOS-NOTTODO/Presentation/Recommend/ViewControllers/RecommendViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  RecommendViewController.swift
+//  iOS-NOTTODO
+//
+//  Created by 김민서 on 2023/02/23.
+//
+
+import UIKit
+
+class RecommendViewController: UIViewController {
+
+    // MARK: - UI Components
+    
+    private lazy var recommendCollectionView = UICollectionView(frame: .zero, collectionViewLayout: layout())
+    let recommendInset: UIEdgeInsets = UIEdgeInsets(top: 0, left: 15, bottom: 0, right: 15)
+    let cellHeight: CGFloat = 137
+    
+    private let navigationView = UIView()
+    private let dismissButton = UIButton()
+    private let navigationTitle = UILabel()
+    private let seperateView = UIView()
+    
+    private var addActionButton = UIButton()
+    
+    // test
+    
+    var recommendList: [RecommendModel] = [
+        RecommendModel(tag: "업무 시간 중", title: "유튜브 보지 않기", body: "유튜브를 보지 않는 것이 당신의 일상에\n어떠한 변화를 일으킬까요?\n행복한 중독 해소를 위해 제안해요!", image: "img_youtube"),
+        RecommendModel(tag: "취침 전", title: "커피 마시지 않기", body: "한국인들은 평균 2잔의 커피를 마신대요.\n적당한 섭취를 위해 제안해요!", image: "img_coffee"),
+        RecommendModel(tag: "기상 직후", title: "SNS 접속하지 않도록 하기", body: "도파민 중독에서 벗어나지 못하는 당신,\n이제는 건강한 삶을 위해 절제해봐요.\n중독 해소를 위한 건강한 제안!", image: "img_insta"),
+        RecommendModel(tag: "업무 시간 중", title: "유튜브 보지 않기", body: "유튜브를 보지 않는 것이 당신의 일상에\n어떠한 변화를 일으킬까요?\n행복한 중독 해소를 위해 제안해요!", image: "img_youtube"),
+        RecommendModel(tag: "취침 전", title: "커피 마시지 않기", body: "한국인들은 평균 2잔의 커피를 마신대요.\n적당한 섭취를 위해 제안해요!", image: "img_coffee"),
+        RecommendModel(tag: "기상 직후", title: "SNS 접속하지 않도록 하기", body: "도파민 중독에서 벗어나지 못하는 당신,\n이제는 건강한 삶을 위해 절제해봐요.\n중독 해소를 위한 건강한 제안!", image: "img_insta")
+    ]
+    
+    // MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUI()
+        setLayout()
+        register()
+        setDelegate()
+    }
+}
+
+// MARK: - Methods
+
+extension RecommendViewController {
+    private func setUI() {
+        view.backgroundColor = .ntdBlack
+        seperateView.backgroundColor = .gray2
+        recommendCollectionView.backgroundColor = .clear
+        
+        dismissButton.do {
+            $0.setBackgroundImage(.delete, for: .normal)
+            // $0.addTarget(self, action: #selector(self.dismissViewController), for: .touchUpInside)
+        }
+        
+        navigationTitle.do {
+            $0.font = .Pretendard(.semiBold, size: 18)
+            $0.textColor = .white
+            $0.text = I18N.recommendNavTitle
+        }
```

Comment:
extension 내부의 프로퍼티나 메소드의 접근 지정자가 모두 동일하다면 `[접근지정자] extension` 처럼 적어줄 수도 있습니다!

```suggestion
private extension RecommendViewController {
   func setUI() {
        view.backgroundColor = .ntdBlack
        seperateView.backgroundColor = .gray2
        recommendCollectionView.backgroundColor = .clear
        
        dismissButton.do {
            $0.setBackgroundImage(.delete, for: .normal)
            // $0.addTarget(self, action: #selector(self.dismissViewController), for: .touchUpInside)
        }
        
        navigationTitle.do {
            $0.font = .Pretendard(.semiBold, size: 18)
            $0.textColor = .white
            $0.text = I18N.recommendNavTitle
        }
```

Reply:
오 이 부분은 처음 알았네요!!! 반영하겠습니당